app/controllers/concerns/dashboard_alerts.rb

```diff
@@ -0,0 +1,16 @@
+module DashboardAlerts
+  extend ActiveSupport::Concern
+
+  def setup_dashboard_alerts
+    @dashboard_alerts = @school.latest_dashboard_alerts.includes(:content_version, :find_out_more).teacher_dashboard.sample(3).map do |dashboard_alert|
```

Comment:
Pass scope through as parameter and school too

Reply:
pass through @school.latest_dashboard_alerts.teacher_dashboard

instead of school

Reply:
return values rather than set instance variables